docs/reference/glossary.asciidoc

```diff
@@ -518,6 +518,15 @@ in the <<glossary-mapping,mapping>>.
 // end::routing-def[]
 --
 
+[[glossary-runtime-fields]] runtime field ::
+// tag::runtime-fields-def[]
+A runtime field is a field that is evaluated at query time. You access runtime
+fields from the search API like any other field, and {es} sees runtime fields
+no differently. You can define runtime fields in the
+<<runtime-mapping-fields,index mapping>> or in the
+<<runtime-search-request,search request>>.
```

Comment:
We reuse these glossary defs in the [Stack
Glossary](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html).
Aside from anchors starting with `glossary-*`, internal xrefs will break when
reused there. If you use the external link syntax, it should work both here and
in the Stack Glossary.
```suggestion
{ref}/runtime-mapping-fields.html[index mapping] or in the
{ref}/runtime-search-request.html[search request].
```

Reply:
Thank you @jrodewig! I'll remember for the future 💭